nimrod/report/report_analysis.py

```diff
@@ -0,0 +1,170 @@
+import re
+import os
+
+
+class Report_Analysis:
+
+    def __init__(self, randoop_original, randoop_modified):
+        self.randoop_original = randoop_original
+        self.randoop_modified = randoop_modified
+
+    def start_analysis(self):
+        left_randoop_original = self.randoop_original[0][2]
+        left_randoop_modified = self.randoop_modified[0][2]
+
+        if os.path.isdir(left_randoop_original) and os.path.isdir(left_randoop_modified):
```

Comment:
I'd suggest also check whether the report test suite files are also available in these directories.
In case they aren't, an exception might be thrown.
I know that you do something like this when trying to open, for example, the file `methods_report.csv` on line 40.
In case a problem happens, an exception is thrown.
But this last exception should be thrown when reading the file, not because is not available.